Rakefile

```diff
@@ -0,0 +1,9 @@
+require 'rake/testtask'
+
+Rake::TestTask.new do |t|
+  t.libs = ["lib"]
+  t.warning = true
+  t.test_files = FileList['specs/*_spec.rb']
```

Comment:
Note the Rakefile is looking for a `specs` folder, but you named it `spec` (singular).